src/main/java/org/web3j/tx/HighThroughputTransactionManager.java

```diff
@@ -0,0 +1,127 @@
+package org.web3j.tx;
+
+import org.web3j.crypto.Credentials;
+import org.web3j.protocol.Web3j;
+import org.web3j.protocol.core.DefaultBlockParameterName;
+import org.web3j.protocol.core.methods.request.RawTransaction;
+import org.web3j.protocol.core.methods.response.EthBlock;
+import org.web3j.protocol.core.methods.response.EthSendTransaction;
+import org.web3j.protocol.core.methods.response.TransactionReceipt;
+import rx.Subscription;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.concurrent.ExecutionException;
+
+/**
+ * High Throughput transaction manager.
+ */
+
+public class HighThroughputTransactionManager extends RawTransactionManager {
```

Comment:
Please could you include an integration tests that demonstrates how to use this class - I'm still not completely clear on the advantages of this approach. An integration test will help clarify.

See https://github.com/web3j/web3j/blob/master/src/integration-test/java/org/web3j/protocol/scenarios/FastRawTransactionManagerIT.java for an example.